### ISSUE TITLE
Swipe animation doesn't match swipe direction

### ISSUE THREAD

**Issue by nic0lette** [stom79/mastalab#441]
When looking at media, one must swipe "up" to dismiss it, but after the swipe gesture is registered, the media then animates down off the screen.
---

Instance: <any>

Mastalab version: 1.8.8

Android version: Android P

**Comment by stom79** [comment]
Do you mean that the threshold to close the media is too important? Because if it is not reached, the media will animate down without closing.

**Comment by nic0lette** [comment]
No, actually the threshold is met, and the media is dismissed. The problem is that I flick it up to dismiss it, and then, a moment after it disappears, it *reappears*, and then animates *down* off the screen.

**Comment by stom79** [comment]
Does it happen for every media or only videos?

**Comment by nic0lette** [comment]
I see it happening with pictures at least.
- Post with embedded image (https://dragon.style/media/UQFQFw4LimufXIAyqQo is an example, boosted by @anthracite@dragon.style)
- Touch image in Mastalab (Image opens)
- Swipe up (animates up and out)
- Image appears again, and immediately animates out but down this time.

**Comment by stom79** [comment]
It's like if the media activity was started twice. Is that issue? I can open twice the media activity (by clicking twice quickly) but the behavior is not like you described.

**Comment by nic0lette** [comment]
No, I touch the media once. I look at it, then I flick "up".
https://imgur.com/wYVjOig
